docs/pom.md

```diff
@@ -0,0 +1,30 @@
+The following metadata fields can be extracted from a pom.xml file
+
+| Metadata category       | Metadata Field                 | Value    |
+|-------------------------|--------------------------------|----------|
+| **authors**       |  authors.name  |     developer.name                            |
+| **authors**       |  authors.email  |    developer.email                            |
+| **authors**       |  author.url  |     developer.url                            |
+| **authors**       |  author.organization |     developer.organization                          |
+| **authors**       |  author.type |     Agent                         |
+| **version** |          version                        |
+| **requirements** |   requirements.value   | groupId.arfifactId                          |
+| **requirements** |   requirements.name   |  arfifactId                        |
+| **requirements** |   requirements.version   | version                       |
+| **requirements** |   requirements.type  | Software_application                        |
+| **package_id** |   package_id.value   | group_id.artifact_id                         |
+| **package_id** |    package_id.type  | String                       |
+| **versiond** |    version.value  | version                    |
+| **versiond** |    version.type  | Release                     |
+| **has_package_field**         |   pom.xml |
+| **issue_tracker**  |     issue_tracker.value    | issue_tracker                      |
+| **issue_tracker**  |     issue_tracker.type     | Url      |
+| **package_distribution**        |   package_distribution.value  | scm_url, repositories(id, name, url)|
+| **package_distribution**        |   packafe_distribution.type | Url|
+| **homepage**         | homepage.value | homepage |
+| **homepage**         | homepage.type | Url|
+| **runtime_platform**     |   runtime_platform.value |  version  |
+| **runtime_platform**     |   runtime_platform.name |   name   |
+| **runtime_platform**     |   runtime_platform.type |   String   |
+
```

Comment:
This file is also missing a full example: the original file, and the link to the somef representation